### ISSUE TITLE
ui: metric settings in the metric creation view

### ISSUE THREAD

**Issue by vkaverin** [VKCOM/statshouse#216]
Currently it's impossible to setup metric in creation view (`/admin/create`) properly. The only thing that's allowed is to give metric a name, but you can't setup desired keys set at the same moment. So, right after the metric was created it's necessary to go back to the `/edit` view and tune the one. This extra step is redundant, because any good metric would need keys tuning.

It would be really nice to have ability to setup all necessary settings at the moment on creation.

Basically, this issue can be summarized as "Please, copy all settings fields from `/edit` to `/admin/create` view".

<img width="1039" alt="image" src="https://user-images.githubusercontent.com/3457402/214616830-e2cde0fa-32af-4156-aaba-a267a2a969d6.png">

**Comment by flyingmutant** [comment]
Thanks for reporting, I'll close this one as it duplicates #194.
